ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-ref.yaml

```diff
@@ -14,8 +14,10 @@ ref:
     - name: BASE_DOMAIN
     - name: CONFIG_TYPE
       default: "minimal"
-    - name: KURYR_ENABLED
```

Comment:
is it safe to remove KURYR_ENABLED ?

Reply:
I am pretty sure. but I will check. I don't think is used anywhere.

Reply:
ha good point. looks liek it has been used. we can live it there I guess.

Reply:
The KURYR_ENABLED variable is replaced by NETWORK_TYPE, 
i.e. KURYR_ENABLED=true is the same things as NETWORK_TYPE=Kuryr, 
but there is a minimal danger so lets move the removal to a separate commit.